### PR TITLE
I translate 66lines - 96lines

### PR DESCRIPTION
**Pull request by SophiaOrekhova**

### CODE REVIEW

**Comment by eooeo** [comment]
We are ivan'$ help Waiting...

**Comment by ieorekhov** [comment]
i'm here, relax your zgopi

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 1]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
 66
 00:01:58,870 --> 00:02:00,740
 Если мы просто вернём всё взад...
-
+If we just to bring everything the back...
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:280]: If we just bring everything back

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 2]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 67
 00:02:00,740 --> 00:02:02,360
 ...Лич сможет заново всё уничтожить!
-
+...Lich can destroy everything again!
 68
 00:02:02,360 --> 00:02:03,360
 Точно...
-
+Yea...
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:288]: Yeah (man, r u srs?)

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 3]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
-
+Erm-m-m...
 73
 00:02:11,120 --> 00:02:12,070
 Придумал!
-
+Invented!
 74
 00:02:12,390 --> 00:02:13,760
 Желаю...
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:311]: I wish

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 4]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 72
 00:02:08,620 --> 00:02:10,340
 Хм-м-м....
-
+Erm-m-m...
 73
 00:02:11,120 --> 00:02:12,070
 Придумал!
-
+Invented!
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:308]: I came up with! (придумать - come up with...)

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 5]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 74
 00:02:12,390 --> 00:02:13,760
 Желаю...
-
+I want...
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:312]: I wish

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 6]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 76
 00:02:17,230 --> 00:02:17,990
 ...никогда...
-
+...never...
 77
 00:02:18,550 --> 00:02:20,700
 ...и не существовал!
-
+... never be a life!
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:324]: never existed

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 7]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 78
 00:02:24,100 --> 00:02:24,680
 Мама?
-
+Mother?
 79
 00:02:25,100 --> 00:02:26,140
 Мама, что случилось?
-
+Mother, what happend?
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:332]:
```suggestion
Mother, what happend?
sd
```

Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:332]: ~~happend~~ happened

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 8]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 81
 00:02:28,420 --> 00:02:29,630
 Это очень важно!
-
+It's very important!
 82
 00:02:30,420 --> 00:02:34,140
 Отведи сегодня мула на базар и продай его как можно дороже!
-
+Take the mule to the bazaar today and sell it as much as you can!
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:344]: ~~sell it as much as you can~~ sell to the highest bidder

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 9]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 83
 00:02:34,300 --> 00:02:34,840
 Что?!
-
+What?!
 84
 00:02:34,990 --> 00:02:36,310
 Продать Бартрума?!
-
+To sell Bartruma?!
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:352]: ~~To sell Bartruma?!~~ Sell Bartrum?

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 10]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 94
 00:02:47,500 --> 00:02:48,580
 Банда Рока?!
-
+Banda Rocka?!
 95
 00:02:48,860 --> 00:02:51,440
 Финн, пожалуйста! Всё будет хорошо, обещаю!
-
+Finn, please! All will be well, i promise!
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:396]: ~~All will be well, i promise!~~ Everything is gonna be fine, I promise!

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 11]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
 66
 00:01:58,870 --> 00:02:00,740
 Если мы просто вернём всё взад...
-
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:280]: sdfsdf

File: s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt [hunk 12]
```diff
@@ -277,123 +277,123 @@ Don’t get excited! Think about it...
...
 88
 00:02:39,830 --> 00:02:40,520
 Пожалуйста!
-
+Please!
 89
 00:02:40,520 --> 00:02:41,760
 Сделай, как я сказала!
-
+Did it, how i said!
```
Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:372]:
~~~Did it how I said~~~ Do it as I tell you

Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:372]:
~~~Did it how I said~~~ Do it as I tell you

Comment (ieorekhov) [inline s5/S05.E01-E02.Finn.the.Human.-.Jake.the.Dog.[rus].srt:372]:
~~~Did it how I said~~~ Do it as I tell you

**Pull request merged**